.github/workflows/ci.yml

```diff
@@ -30,9 +30,7 @@ jobs:
           node-version-file: .nvmrc
           cache: 'npm'
       - name: Install dependencies
-        run: |
-          npm ci
-          npx playwright install-deps
+        run: npm ci
```

Comment:
This would have no longer worked, but it shouldn't be needed since `@brightspace-ui/testing` does it for us.